project/Component/NefModels/CarbonModel.swift

```diff
@@ -121,16 +121,16 @@ extension CarbonStyle.Color {
 public struct CarbonError: Error {
     public let filename: String
     public let snippet: String
-    public let error: CarbonErrorOption
+    public let error: CarbonError.Option
     
-    public init(filename: String, snippet: String, error: CarbonErrorOption) {
+    public init(filename: String, snippet: String, error: CarbonError.Option) {
```

Comment:
Suggestion: Rename `Option` to `Cause`, I think it is more meaningful. Likewise, the `error` field should be renamed to `cause`.

Reply:
I like it :)